### PR TITLE
ZEPPELIN-3162. Fixed Checkstyle issues in neo4j module

### PR DESCRIPTION
**Pull request by HorizonNet**
### What is this PR for?
Fixed the Checkstyle errors and warnings in the neo4j module.

### What type of PR is it?
Improvement

### Todos
* [ ] - Task

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-3162

### How should this be tested?
* CI pass

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

File: neo4j/src/main/java/org/apache/zeppelin/graph/neo4j/Neo4jCypherInterpreter.java [hunk 1]
```diff
@@ -205,7 +206,9 @@ private void addValueToLine(String key, List<String> columns, List<String> line,
       if (value instanceof Collection) {
         try {
           value = jsonMapper.writer().writeValueAsString(value);
-        } catch (Exception ignored) {}
+        } catch (Exception ignored) {
+          logger.info(ignored.getMessage());
```
Comment (felixcheung) [inline neo4j/src/main/java/org/apache/zeppelin/graph/neo4j/Neo4jCypherInterpreter.java]: might be confusing to log an exception message (that is ignored)?

Comment (HorizonNet) [reply neo4j/src/main/java/org/apache/zeppelin/graph/neo4j/Neo4jCypherInterpreter.java]: The alternative would be to put it onto the ignore list. Currently it seems that there are no items on this list so far. Question would be where to put the ignore file? Into the `_tools` directory?

Comment (felixcheung) [reply neo4j/src/main/java/org/apache/zeppelin/graph/neo4j/Neo4jCypherInterpreter.java]: ignore file for checkstyle?
let's change this to `logger.debug("ignored exception: " + e.getMessage());` (rename the variable)

**Comment by felixcheung** [comment]
seems like test fail for unrelated reason - is it out of date or something?
```
[WARNING] The requested profile "hadoop2" could not be activated because it does not exist.
[ERROR] Failed to execute goal com.github.eirslett:frontend-maven-plugin:1.3:npm (npm test) on project zeppelin-web: Failed to run task: 'npm run test' failed. (error code 1) -> [Help 1]
```

**Comment by HorizonNet** [comment]
I merged the latest master into this branch. Let's wait for the build.

**Comment by HorizonNet** [comment]
Reverted `zeppelin-web/package-lock.json` to the one from the `master` branch.

**Comment by felixcheung** [comment]
one test failed, could you check?

**Comment by HorizonNet** [comment]
I'm currently re-running all the failing PRs.

**Comment by felixcheung** [comment]
merging if no more comment

**Pull request closed without merging**